client/components/ClientOnly/ClientOnly.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Rationale for this is that using `localStorage` in a feature prevents us from server-side rendering it. The `useLocalStorage` hook checks that the context rendered here is present, which makes it impossible to accidentally use browser-specific features during SSR.